kvbc/include/kv_types.hpp

```diff
@@ -10,32 +10,42 @@
 // file.
 //
 
-#ifndef CONCORD_BFT_UTIL_KV_TYPES_H_
-#define CONCORD_BFT_UTIL_KV_TYPES_H_
+#pragma once
 
+#include <string_view>
```

Comment:
Why do we need the `<string_view>` include here?

Reply:
check out the hash

Reply:
Thanks, missed that.